zsh/.zshrc

```diff
@@ -94,7 +94,15 @@ export LSCOLORS='Exfxcxdxbxegedabagacad'
 export LS_COLORS='di=1;34;40:ln=35;40:so=32;40:pi=33;40:ex=31;40:bd=34;46:cd=34;43:su=0;41:sg=0;46:tw=0;42:ow=0;43:'
 
 load-our-ssh-keys() {
-  eval `ssh-agent -s` &> /dev/null
+  if [ -z "$SSH_AUTH_SOCK" ]; then
+   # Check for a currently running instance of the agent
+   RUNNING_AGENT="`ps -ax | grep 'ssh-agent -s' | grep -v grep | wc -l | tr -d '[:space:]'`"
```

Comment:
I'd prefer `$(command)` instead of backticks, but otherwise this looks good.

Reply:
And yes, there are other places I need to fix that :-/

Reply:
Updated PR.